### PR TITLE
chore(nuxt): migrate to nuxtjs/i18n

### PR DESCRIPTION
**Pull request by MrTimscampi**
The name of the nuxt-i18n module changed to @nuxtjs/i18n with the release of v7.

This PR migrates us to that new version.

### CODE REVIEW

**Review by ThibaultNocchi — APPROVED** [review]

File: package-lock.json [hunk 1]
```diff
@@ -25559,7 +25578,7 @@
         },
         "browserify-aes": {
           "version": "1.2.0",
-          "resolved": "https://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
+          "resolved": "http://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
```
Comment (ferferga) [inline package-lock.json:25581]:
```suggestion
          "resolved": "https://registry.npmjs.org/browserify-aes/-/browserify-aes-1.2.0.tgz",
```

File: package-lock.json [hunk 2]
```diff
@@ -25593,7 +25612,7 @@
         },
         "browserify-rsa": {
           "version": "4.0.1",
-          "resolved": "https://registry.npmjs.org/browserify-rsa/-/browserify-rsa-4.0.1.tgz",
+          "resolved": "http://registry.npmjs.org/browserify-rsa/-/browserify-rsa-4.0.1.tgz",
```
Comment (ferferga) [inline package-lock.json:25615]:
```suggestion
          "resolved": "https://registry.npmjs.org/browserify-rsa/-/browserify-rsa-4.0.1.tgz",
```

File: package-lock.json [hunk 3]
```diff
@@ -25624,7 +25643,7 @@
         },
         "buffer": {
           "version": "4.9.1",
-          "resolved": "https://registry.npmjs.org/buffer/-/buffer-4.9.1.tgz",
+          "resolved": "http://registry.npmjs.org/buffer/-/buffer-4.9.1.tgz",
```
Comment (ferferga) [inline package-lock.json:25646]:
```suggestion
          "resolved": "https://registry.npmjs.org/buffer/-/buffer-4.9.1.tgz",
```

File: package-lock.json [hunk 4]
```diff
@@ -25760,7 +25779,7 @@
         },
         "chalk": {
           "version": "1.1.3",
-          "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
+          "resolved": "http://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```
Comment (ferferga) [inline package-lock.json:25782]:
```suggestion
          "resolved": "https://registry.npmjs.org/chalk/-/chalk-1.1.3.tgz",
```

File: package-lock.json [hunk 5]
```diff
@@ -26007,7 +26026,7 @@
         },
         "create-hash": {
           "version": "1.2.0",
-          "resolved": "https://registry.npmjs.org/create-hash/-/create-hash-1.2.0.tgz",
+          "resolved": "http://registry.npmjs.org/create-hash/-/create-hash-1.2.0.tgz",
```
Comment (ferferga) [inline package-lock.json:26029]:
```suggestion
          "resolved": "https://registry.npmjs.org/create-hash/-/create-hash-1.2.0.tgz",
```

File: package-lock.json [hunk 6]
```diff
@@ -26019,7 +26038,7 @@
         },
         "create-hmac": {
           "version": "1.1.7",
-          "resolved": "https://registry.npmjs.org/create-hmac/-/create-hmac-1.1.7.tgz",
+          "resolved": "http://registry.npmjs.org/create-hmac/-/create-hmac-1.1.7.tgz",
```
Comment (ferferga) [inline package-lock.json:26041]:
```suggestion
          "resolved": "https://registry.npmjs.org/create-hmac/-/create-hmac-1.1.7.tgz",
```

File: package-lock.json [hunk 7]
```diff
@@ -26226,7 +26245,7 @@
         },
         "diffie-hellman": {
           "version": "5.0.3",
-          "resolved": "https://registry.npmjs.org/diffie-hellman/-/diffie-hellman-5.0.3.tgz",
+          "resolved": "http://registry.npmjs.org/diffie-hellman/-/diffie-hellman-5.0.3.tgz",
```
Comment (ferferga) [inline package-lock.json:26248]:
```suggestion
          "resolved": "https://registry.npmjs.org/diffie-hellman/-/diffie-hellman-5.0.3.tgz",
```

File: package-lock.json [hunk 8]
```diff
@@ -27638,7 +27657,7 @@
           "dependencies": {
             "minimist": {
               "version": "1.2.5",
-              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
+              "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
```
Comment (ferferga) [inline package-lock.json:27660]:
```suggestion
              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.5.tgz",
```

File: package-lock.json [hunk 9]
```diff
@@ -27797,7 +27816,7 @@
         },
         "http-errors": {
           "version": "1.6.3",
-          "resolved": "https://registry.npmjs.org/http-errors/-/http-errors-1.6.3.tgz",
+          "resolved": "http://registry.npmjs.org/http-errors/-/http-errors-1.6.3.tgz",
```
Comment (ferferga) [inline package-lock.json:27819]:
```suggestion
          "resolved": "https://registry.npmjs.org/http-errors/-/http-errors-1.6.3.tgz",
```

File: package-lock.json [hunk 10]
```diff
@@ -28332,7 +28351,7 @@
           "dependencies": {
             "minimist": {
               "version": "1.2.0",
-              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+              "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```
Comment (ferferga) [inline package-lock.json:28354]:
```suggestion
              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```

File: package-lock.json [hunk 11]
```diff
@@ -28363,7 +28382,7 @@
         },
         "karma-webpack": {
           "version": "2.0.13",
-          "resolved": "https://registry.npmjs.org/karma-webpack/-/karma-webpack-2.0.13.tgz",
+          "resolved": "http://registry.npmjs.org/karma-webpack/-/karma-webpack-2.0.13.tgz",
```
Comment (ferferga) [inline package-lock.json:28385]:
```suggestion
          "resolved": "https://registry.npmjs.org/karma-webpack/-/karma-webpack-2.0.13.tgz",
```

File: package-lock.json [hunk 12]
```diff
@@ -28527,7 +28546,7 @@
         },
         "log4js": {
           "version": "0.6.38",
-          "resolved": "https://registry.npmjs.org/log4js/-/log4js-0.6.38.tgz",
+          "resolved": "http://registry.npmjs.org/log4js/-/log4js-0.6.38.tgz",
```
Comment (ferferga) [inline package-lock.json:28549]:
```suggestion
          "resolved": "https://registry.npmjs.org/log4js/-/log4js-0.6.38.tgz",
```

File: package-lock.json [hunk 13]
```diff
@@ -28541,7 +28560,7 @@
             },
             "readable-stream": {
               "version": "1.0.34",
-              "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
+              "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
```
Comment (ferferga) [inline package-lock.json:28563]:
```suggestion
              "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-1.0.34.tgz",
```

File: package-lock.json [hunk 14]
```diff
@@ -28700,7 +28719,7 @@
             },
             "minimist": {
               "version": "1.2.0",
-              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
+              "resolved": "http://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```
Comment (ferferga) [inline package-lock.json:28722]:
```suggestion
              "resolved": "https://registry.npmjs.org/minimist/-/minimist-1.2.0.tgz",
```

File: package-lock.json [hunk 15]
```diff
@@ -28827,7 +28846,7 @@
         },
         "minimist": {
           "version": "0.0.8",
-          "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
+          "resolved": "http://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
```
Comment (ferferga) [inline package-lock.json:28849]:
```suggestion
          "resolved": "https://registry.npmjs.org/minimist/-/minimist-0.0.8.tgz",
```

File: package-lock.json [hunk 16]
```diff
@@ -28851,7 +28870,7 @@
         },
         "mkdirp": {
           "version": "0.5.1",
-          "resolved": "https://registry.npmjs.org/mkdirp/-/mkdirp-0.5.1.tgz",
+          "resolved": "http://registry.npmjs.org/mkdirp/-/mkdirp-0.5.1.tgz",
```
Comment (ferferga) [inline package-lock.json:28873]:
```suggestion
          "resolved": "https://registry.npmjs.org/mkdirp/-/mkdirp-0.5.1.tgz",
```

File: package-lock.json [hunk 17]
```diff
@@ -29165,7 +29184,7 @@
         },
         "onetime": {
           "version": "1.1.0",
-          "resolved": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
+          "resolved": "http://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
```
Comment (ferferga) [inline package-lock.json:29187]:
```suggestion
          "resolved": "https://registry.npmjs.org/onetime/-/onetime-1.1.0.tgz",
```

File: package-lock.json [hunk 18]
```diff
@@ -29270,7 +29289,7 @@
         },
         "parse-asn1": {
           "version": "5.1.1",
-          "resolved": "https://registry.npmjs.org/parse-asn1/-/parse-asn1-5.1.1.tgz",
+          "resolved": "http://registry.npmjs.org/parse-asn1/-/parse-asn1-5.1.1.tgz",
```
Comment (ferferga) [inline package-lock.json:29292]:
```suggestion
          "resolved": "https://registry.npmjs.org/parse-asn1/-/parse-asn1-5.1.1.tgz",
```

File: package-lock.json [hunk 19]
```diff
@@ -29493,7 +29512,7 @@
         },
         "public-encrypt": {
           "version": "4.0.2",
-          "resolved": "https://registry.npmjs.org/public-encrypt/-/public-encrypt-4.0.2.tgz",
+          "resolved": "http://registry.npmjs.org/public-encrypt/-/public-encrypt-4.0.2.tgz",
```
Comment (ferferga) [inline package-lock.json:29515]:
```suggestion
          "resolved": "https://registry.npmjs.org/public-encrypt/-/public-encrypt-4.0.2.tgz",
```

File: package-lock.json [hunk 20]
```diff
@@ -29609,7 +29628,7 @@
         },
         "readable-stream": {
           "version": "2.3.6",
-          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
+          "resolved": "http://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```
Comment (ferferga) [inline package-lock.json:29631]:
```suggestion
          "resolved": "https://registry.npmjs.org/readable-stream/-/readable-stream-2.3.6.tgz",
```

File: package-lock.json [hunk 21]
```diff
@@ -29922,7 +29941,7 @@
         },
         "sha.js": {
           "version": "2.4.11",
-          "resolved": "https://registry.npmjs.org/sha.js/-/sha.js-2.4.11.tgz",
+          "resolved": "http://registry.npmjs.org/sha.js/-/sha.js-2.4.11.tgz",
```
Comment (ferferga) [inline package-lock.json:29944]:
```suggestion
          "resolved": "https://registry.npmjs.org/sha.js/-/sha.js-2.4.11.tgz",
```

File: package-lock.json [hunk 22]
```diff
@@ -30388,7 +30407,7 @@
         },
         "table": {
           "version": "3.8.3",
-          "resolved": "https://registry.npmjs.org/table/-/table-3.8.3.tgz",
+          "resolved": "http://registry.npmjs.org/table/-/table-3.8.3.tgz",
```
Comment (ferferga) [inline package-lock.json:30410]:
```suggestion
          "resolved": "https://registry.npmjs.org/table/-/table-3.8.3.tgz",
```

File: package-lock.json [hunk 23]
```diff
@@ -30445,7 +30464,7 @@
         },
         "through": {
           "version": "2.3.8",
-          "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz",
+          "resolved": "http://registry.npmjs.org/through/-/through-2.3.8.tgz",
```
Comment (ferferga) [inline package-lock.json:30467]:
```suggestion
          "resolved": "https://registry.npmjs.org/through/-/through-2.3.8.tgz",
```

File: package-lock.json [hunk 24]
```diff
@@ -31263,7 +31282,7 @@
         },
         "wrap-ansi": {
           "version": "2.1.0",
-          "resolved": "https://registry.npmjs.org/wrap-ansi/-/wrap-ansi-2.1.0.tgz",
+          "resolved": "http://registry.npmjs.org/wrap-ansi/-/wrap-ansi-2.1.0.tgz",
```
Comment (ferferga) [inline package-lock.json:31285]:
```suggestion
          "resolved": "https://registry.npmjs.org/wrap-ansi/-/wrap-ansi-2.1.0.tgz",
```

**Review by ferferga — CHANGES_REQUESTED** [review]
Are you on latest npm 6.x branch and cleared cache? Never had this issue since a few npm updates.

**Comment by MrTimscampi** [comment]
@ferferga NPM 6.14.14, cache completely removed, node_modules completely removed, still happens.

As far as I can tell from looking at upstream issues, it's an intermittent NPM issue that's been happening for years, so I don't think blocking things over this or making PRs constantly to fix it is really a good call, since it'll happen anyway (Dependabot did it too a few times in the past, for example).

**Pull request merged**